### PR TITLE
feat(indexer): Indexing Oracles Data

### PR DESCRIPTION
**Pull request by nichellekoh**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Reverted Changes in PR https://github.com/DeFiCh/whale/pull/749
Update time intervals according to oracles feed and requirements - 15 mins, 1 hour, 1 day

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: src/module.indexer/model/dftx/oracle.e2e.ts [hunk 1]
```diff
@@ -366,3 +367,68 @@ describe('invalidate set oracle data', () => {
...
+    ], {
+      weightage: 1
+    })
+    await container.generate(1)
+
+    const oneMinute = 60
+    let mockTime = Math.floor(new Date().getTime() / 1000)
+    for (let h = 0; h < 24; h++) {
+      for (let z = 0; z < 4; z++) {
+        mockTime += (z * 15) * oneMinute
```
Comment (joeldavidw) [inline src/module.indexer/model/dftx/oracle.e2e.ts:404]:
```suggestion
        mockTime += 15 * oneMinute
```

Comment (joeldavidw) [inline src/module.indexer/model/dftx/oracle.e2e.ts:404]:
```suggestion
        mockTime += (15 * oneMinute) + 1 // +1 sec to fall into the next 15 mins bucket```

File: src/module.indexer/model/dftx/oracle.e2e.ts [hunk 2]
```diff
@@ -366,3 +367,68 @@ describe('invalidate set oracle data', () => {
...
+        await client.misc.setMockTime(mockTime)
+        await container.generate(1)
```
Comment (joeldavidw) [inline src/module.indexer/model/dftx/oracle.e2e.ts:406]:
```suggestion
        await container.generate(11)
```
Maybe we should wait for 11 blocks. For the median time to change.

Comment (joeldavidw) [reply src/module.indexer/model/dftx/oracle.e2e.ts:406]: 1 Block:
```
 medianTime: 1647336701,
time: 1647337602
```
11 Blocks:
```
medianTime: 1647338104,
time: 1647338104
```

Comment (joeldavidw) [reply src/module.indexer/model/dftx/oracle.e2e.ts:406]: However, waiting for 11 blocks does add into the test time.

Comment (joeldavidw) [reply src/module.indexer/model/dftx/oracle.e2e.ts:406]: For regtest 2 blocks is enough.
```
await container.generate(2)
```

**Review by joeldavidw — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**